tools/ci_build/github/linux/docker/scripts/install_openvino.sh

```diff
@@ -7,13 +7,14 @@ o) OPENVINO_VERSION=${OPTARG};;
 esac
 done
 
-OPENVINO_VERSION=${OPENVINO_VERSION:=2020.2}
-export INTEL_OPENVINO_DIR=/data/dldt/openvino_${OPENVINO_VERSION}
-git clone https://github.com/opencv/dldt.git ${INTEL_OPENVINO_DIR}
+OPENVINO_VERSION=${OPENVINO_VERSION:=2020.4}
+export INTEL_OPENVINO_DIR=/data/openvino/openvino_${OPENVINO_VERSION}.287
```

Comment:
>287 [](start = 70, length = 3)

should we make the minor version as a parameter also? so we don't need to always change this file.